Sources/PointFree/PageLayout.swift

```diff
@@ -51,36 +63,39 @@ func respond<A, B>(
 
     return { conn in
       let newLayoutData = layoutData(conn.data) |> \.flash .~ conn.request.session.flash
+      let pageLayout = metaLayout(simplePageLayout(view))
+        .map(addGoogleAnalytics)
+        .contramap(
+          Metadata.create(
+            description: newLayoutData.description,
+            image: newLayoutData.image,
+            title: newLayoutData.title,
+            twitterCard: newLayoutData.twitterCard,
+            twitterSite: "@pointfreeco",
+            type: newLayoutData.openGraphType,
+            url: newLayoutData.currentRoute.map(url(to:))
+          )
+      )
 
       return conn
         |> writeSessionCookieMiddleware(\.flash .~ nil)
         >-> respond(
-          body: simplePageLayout(view).rendered(with: newLayoutData),
+          body: pageLayout.rendered(with: newLayoutData),
           contentType: .html
       )
     }
 }
 
 func simplePageLayout<A>(_ contentView: View<A>) -> View<SimplePageLayoutData<A>> {
   return View { layoutData in
-    return document([
+    document([
       html([
         head([
           title(layoutData.title),
           style(renderedNormalizeCss),
           style(styleguide),
           style(layoutData.extraStyles),
           meta(viewport: .width(.deviceWidth), .initialScale(1)),
-          script(
```

Comment:
Get this for free with `addGoogleAnalytics` :)

Reply:
oh ha, I was actually going to deprecate `addGoogleAnalytics` in favor of just putting the script in directly.

Reply:
lol. either way! are you into reusing the meta stuff or should that be direct?

Reply:
Tbh I thought we could add the site `id` as a configuration option to `addGoogleAnalytics` and it's a nice example of an HTML `Node` transformer.

Reply:
I think that could go direct too since we're putting the fields into the layout data.

someday we should sit down and figure out how/if layouts compose, but maybe for now it's just easily to be very direct.

Reply:
I actually started going with the direct route (see remnants from the second commit in this PR), but ended up reusing the meta stuff just because it illustrates a nice boundary between what could live in swift-web and what can't. Some of that layer is a direct mapping, but others we stub out.

Reply:
oh cool cool, in that case let's leave as is!